packages/cli/src/runDeploy.ts

```diff
@@ -133,6 +133,7 @@ export async function runDeploy(opts: DeployOptions): Promise<WorldDeploy> {
         : undefined,
     }),
     account,
+    pollingInterval: 100,
```

Comment:
Reducing polling time for a short time during deployment should be safe and makes a significant impact on deployment time (with the project i was testing with this change brought the time from 16s to 4s after the previous improvements were implemented)

Reply:
(moved this to `waitForTransactions` instead to only increase the polling rate where it matters)

Reply:
Moved this change to #3198 since it's unrelated to library linking, and CI wasn't happy with it yet